src/component-library/TokenInput/TokenInput.tsx

```diff
@@ -46,10 +45,25 @@ const TokenInput = forwardRef<HTMLInputElement, TokenInputProps>(
   ): JSX.Element => {
     const inputRef = useDOMRef(ref);
     const [value, setValue] = useState<number | undefined>(valueProp);
+    const [resize, setResize] = useState({ state: false, position: 0 });
 
     const handleClickBalance = () => triggerChangeEvent(inputRef, balance);
 
+    const handleOverflow: ChangeEventHandler<HTMLInputElement> = (e) => {
+      const el = e.target as HTMLInputElement;
```

Comment:
I think `event` instead of `e` and `element` instead of `el` would be better names here to add more readability.

Reply:
This code was updated